playbook.md

```diff
@@ -1062,98 +1045,35 @@ In general:
-them later.
+While we do everything we can to make sure the client is happy with our solution, we are also mindful of what's practical. We don't do serious bits of development work on tickets, or trial new approaches. We don't play with new tools or sink hours into interesting bugs. We set those things aside, and do them later.
 
-The main purpose of a ticket is to take some action that solves the problem, as
-quickly as possible. Generally speaking, we do the most time-efficient thing
-that we can. Of several potential solutions that solve the problem, assuming
-none is a bad one, we do the one which can be implemented the most quickly.
+The main purpose of a ticket is to take some action that solves the problem, as quickly as possible. Generally speaking, we do the most time-efficient thing that we can. Of several potential solutions that solve the problem, assuming none is a bad one, we do the one which can be implemented the most quickly.
 
 #### Deciding what to work on
 
-You are free to work on whichever of your assigned tickets you think is most
-important. But there are some important things to bear in mind.
+Developers on support are free to work on whichever tickets they are assigned to and think is most important. But there are some important things to bear in mind.
 
 ##### Triage
 
 We all have limited time. We try to spend it wisely. All other things being
 equal, it is better to spend half an hour solving each of four tickets than to
 spend two hours on one issue.
 
-##### Procrastinating
-
-We don't procrastinate about dealing with difficult tickets. Most of our SLA
-breaches happen when a complex ticket arrives and is assigned to a busy person.
-In this situation, it's only natural that we prefer to do other work first. But
-it is vital that we don't let the hard tickets gather dust while we crack
-through the easy ones, so we try to be mindful of this bias.
-
 ##### Priority
 
 The priority of tickets is important, and we must be biased towards dealing with
 more urgent tickets before less urgent ones.
 
-Zendesk can help us to balance these factors. When you log in and land on the
-dashboard, you will see a list of your tickets, ordered by age and priority.
-
-Unless there's a good reason not to, we deal with tickets in the order they are
-displayed. There's a "play" button at the top right hand corner of the ticket
-list that will serve up each ticket in turn, moving to the next as you mark
-things open, pending or solved.
-
-Zendesk knows each ticket's age, the time it's been since it was updated, its
-priority and the time left until we breach our SLA. Generally speaking, it makes
-good decisions about priority.
-
 Each ticket's priority should be reviewed regularly: whenever we update a
 ticket, we check the priority to make sure it's still right.
 
 #### Initial investigation
 
-When we pick up new tickets or are assigned one where no investigation has yet
```

Comment:
Same question: why are we removing this section? Are we moving it to the first line support guidance? I think it's useful to give as much guidance as possible for people who may be new to being assigned support tickets and dealing with them.